Pc 1122 investigation into c# and .net update

Glenn-Clarke opened the pull request:
[Link to Jira ticket](https://beisdigital.atlassian.net/browse/PC-1050)

# Description

C# & .NET update changes with port number 8080 reverted to 80

# Checklist

- [x] I have made any necessary updates to the documentation
- [x] I have checked there are no unnecessary IDE warnings in this PR
- [x] I have checked there are no unintentional line ending changes
- [x] I have added tests where applicable
- [x] If I have made any changes to the code, I have used the IDE auto-formatter on it
- [x] If I have made any changes to website flow, I have checked forward and back behaviour is still consistent
- [x] If I have made any changes to content strings or resource files, I have followed [the documentation](https://github.com/UKGovernmentBEIS/beis-simple-energy-advice-beta/blob/main/docs/LocalisationDocumentation.md)

jdgage commented on docker-compose.yml:
```diff
@@ -7,6 +7,7 @@ services:
       - "5001:80"
     environment:
       ASPNETCORE_ENVIRONMENT: "Development"
+      ASPNETCORE_HTTP_PORTS: "80"
```
Assuming this all works fine running locally?

Glenn-Clarke replied:
I tested this change yesterday after you suggested trying it locally, and it appears to work fine, including PDF generation.

jdgage reviewed (commented):
Could you have a quick look into the (I think new with .NET 8) build warnings? i.e. whether they're something that we can easily deal with now?

Glenn-Clarke commented:
Looks like the build warnings were simple replacements, but I'm testing the changes locally now

jdgage reviewed (dismissed):
Did you look at the warning about lower-cased characters in that migration class name? I think we should be able to just rename the class (and maybe add a comment about why)

Glenn-Clarke commented:
They don't seem to come up in the build... I'll have a look now quickly and see if I can find them

Glenn-Clarke commented:
They didn't show up in my IDE unless I went to file in question 😓 I found them using the github build warnings, and I've fixed it now. I renamed the file and class for the migrations and the data in my local db seems to be fine. Hope this is ok!

Glenn-Clarke commented:
Docker compose up seems to be ok? I think?
